docs/topics/proc_creating-spring-boot-app.adoc

```diff
@@ -0,0 +1,209 @@
+[#proc_creating-a-spring-boot-app]
+= Creating a {SpringBoot} Application
+
+In addition to xref:mission-http-api-spring-boot-tomcat[using a booster], you can also create new {SpringBoot} applications from scratch and deploy them to OpenShift.
+
+== Creating an Application
+
+.Prerequisites
+* `mvn` installed.
+
+.Procedure
+
+. Create the application folder.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mkdir myApp
+$ cd myApp
+----
+
+. Create a `pom.xml`.
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.openshift.booster</groupId>
+    <artifactId>spring-boot-booster-parent</artifactId>
+    <version>1.5.10-15</version>
+  </parent>
+
+  <artifactId>my-app</artifactId>
+  <version>1.5.10-5-SNAPSHOT</version>
+  <groupId>com.example</groupId>
+
+  <name>MyApp</name>
+  <description>My Application</description>
+
+
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-tomcat</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.cxf</groupId>
+      <artifactId>cxf-spring-boot-starter-jaxrs</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.jaxrs</groupId>
+      <artifactId>jackson-jaxrs-json-provider</artifactId>
+    </dependency>
+  </dependencies>
+
+  <profiles>
+    <profile>
```

Comment:
There also needs to be the `openshift` profile, which would configure the Fabric8 Maven plugin.